docker/.editorconfig

```diff
@@ -3,10 +3,15 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
-
-[*]
+indent_style = space
```

Comment:
it is to standardise code style across all developers - e.g. Makefile only accept tabs as indentation, although I personally like spaces. With Atom's editorconfig plugin, this file is read automatically and the setting is applied (otherwise Atom would convert it to spaces)